### ISSUE TITLE
 Invalid grid element

### ISSUE THREAD

**Issue by vscodeerrors** [microsoft/vscode#80614]
Issue Id: <b>3a4b08c2-0cf6-689a-15b7-3b3c7f8253d8</b><br /><br />*Versions* <br />- 1.38.0-insider (9/3/2019 11:14:03 PM)<br />-  5ab84ce18fe7dee304325952f2849264dd4a48c1<br />-  190a29c9f8cb60859adaad497f316ceed717b3c2<br />-  dcc01858d3f75696568eb751238d01b30a1e7d33<br />-  e77b008fb04222192a608906ff603bd72943d779<br />*Stack* <br />Error: Invalid grid element<br />[/vs/base/browser/ui/grid/grid.ts#171:8 (getGridLocation)](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/base/browser/ui/grid/grid.ts#L171:8 (getGridLocation))<br />[/vs/base/browser/ui/grid/grid.ts#180:12 (getGridLocation)](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/base/browser/ui/grid/grid.ts#L180:12 (getGridLocation))<br />[/vs/base/browser/ui/grid/grid.ts#437:9 (getViewLocation)](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/base/browser/ui/grid/grid.ts#L437:9 (getViewLocation))<br />[/vs/base/browser/ui/grid/grid.ts#372:24 (getViewSize)](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/base/browser/ui/grid/grid.ts#L372:24 (getViewSize))<br />[/vs/workbench/browser/parts/editor/editorPart.ts#594:36 (doRestoreGroup)](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/workbench/browser/parts/editor/editorPart.ts#L594:36 (doRestoreGroup))<br />[/vs/workbench/browser/parts/editor/editorPart.ts#330:7 (restoreGroup)](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/workbench/browser/parts/editor/editorPart.ts#L330:7 (restoreGroup))<br />[/vs/workbench/browser/parts/editor/editorGroupView.ts#863:17 (doOpenEditor)](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/workbench/browser/parts/editor/editorGroupView.ts#L863:17 (doOpenEditor))<br />[/vs/workbench/browser/parts/editor/editorGroupView.ts#816:40](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/workbench/browser/parts/editor/editorGroupView.ts#L816:40)<br />    at Generator.next (<anonymous>)<br />[/vs/base/common/errors.ts#199:0](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/base/common/errors.ts#L199:0)<br />    at new Promise (<anonymous>)<br />[/vs/base/common/errors.ts#199:0 (options)](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/base/common/errors.ts#L199:0 (options))<br />[/vs/workbench/browser/parts/editor/editorGroupView.ts#800:39 (openEditor)](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/workbench/browser/parts/editor/editorGroupView.ts#L800:39 (openEditor))<br />[/vs/workbench/services/editor/browser/editorService.ts#279:41](https://github.com/microsoft/vscode/blob/3db7e09f3b61f915d03bbfa58e258d6eee843f35/src/vs/workbench/services/editor/browser/editorService.ts#L279:41)<br />    at Generator.next (<anonymous>)<br />    at out/vs/workbench/workbench.desktop.m<br />

**Comment by joaomoreno** [comment]
@sbatten Isn't this the editor grid?

**Comment by sbatten** [comment]
yes, it is but could be a result of changes to grid serialization

**Comment by joaomoreno** [comment]
Got it. Any thoughts on how to repro?

**Comment by sbatten** [comment]
nope, couldn't repro myself. the number of hits is fairly small imo and no reports from 1.39-insiders

**Comment by joaomoreno** [comment]
Close it up!
